docs/codeql/codeql-language-guides/analyzing-data-flow-in-go.rst

```diff
@@ -166,7 +166,7 @@ Global data flow tracks data flow throughout the entire program, and is therefor
 Using global data flow
 ~~~~~~~~~~~~~~~~~~~~~~
 
-The global data flow library is used by implementing the signature ``DataFlow::ConfigSig`` and applying the module ``DataFlow::Global<ConfigSig>``:
+We can use the global data flow library by implementing the signature ``DataFlow::ConfigSig`` and applying the module ``DataFlow::Global<ConfigSig>``:
```

Comment:
Minor: "applying the module ``DataFlow::Global<ConfigSig>``" has the potential to be confusing if readers don't realise that `ConfigSig` is the expected type of the type parameter / how module instantiation works in QL. Perhaps this could refer to the example below as illustration. 

Perhaps (additionally or alternatively) this could link to https://codeql.github.com/docs/ql-language-reference/modules/#parameterized-modules or similar.